doc/source/installation.rst

```diff
@@ -42,103 +42,20 @@ Here are links to the latest wheels (which are built off of master). To install
 .. _`MacOS Python 3.5`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev2-cp35-cp35m-macosx_10_6_intel.whl
 .. _`MacOS Python 2.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev2-cp27-cp27m-macosx_10_6_intel.whl
 
```

Comment:
Can you also add instructions for installing Ray from any particular recent commit? E.g.,

```suggestion

To install Ray from any recent commit, do the following.

.. code-block:: bash
  RAY_COMMIT=63a6b0e710fded230f340a237e55085fb4ace4e9
  RAY_VERSION=0.8.0.dev3  # You can find this in python/ray/__init__.py.
  RAY_BRANCH=master

  pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_BRANCH/$RAY_COMMIT/ray-$RAY_VERSION-cp36-cp36m-manylinux1_x86_64.whl
```

Actually, the above doesn't quite work because we also need to modify the wheel for Mac/Linux and different Python versions. Maybe it's best to put it in a simple script and then link to it from here.